Pod/Classes/ios/NYTPhotosViewController.m

```diff
@@ -239,8 +240,32 @@ - (void)actionButtonTapped:(id)sender {
                 [self.delegate photosViewController:self actionCompletedWithActivityType:activityType];
             }
         };
-        
-        [self presentViewController:activityViewController animated:YES completion:nil];
+
+        [self displayActivityViewController:activityViewController animated:YES];
+    }
+}
+
+- (void)displayActivityViewController:(UIActivityViewController *)controller animated:(BOOL)animated {
+
+    if (UI_USER_INTERFACE_IDIOM() == UIUserInterfaceIdiomPhone) {
+        // iPhone
```

Comment:
All of these comments seem a bit redundant. The rest of this file tends to avoid comments simply stating what code does in favor of comments explaining why it does it if necessary.